Resources/Textures/Structures/Windows/plastitanium_window.rsi/meta.json

```diff
@@ -0,0 +1,46 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from Paradise",
```

Comment:
cc-by-sa-3 requires us to be more specific. If you could point to the repository directly would suffice.

Reply:
Fixed same with tiles attribution.